Core/include/Acts/Propagator/RiddersPropagator.ipp

```diff
@@ -105,33 +123,23 @@ auto Acts::RiddersPropagator<propagator_t>::propagate(
   // Wiggle each dimension individually
   for (unsigned int i = 0; i < eBoundSize; i++) {
     derivatives[i] =
-        wiggleDimension(opts, start, i, target, nominalParameters, deviations);
+        wiggleParameter(opts, start, i, target,
+                        nominalFinalParameters.parameters(), deviations);
   }
-  if (start.covariance()) {
-    // Test if target is disc - this may lead to inconsistent results
-    if (target.type() == Surface::Disc) {
-      for (const std::vector<BoundVector>& deriv : derivatives) {
-        if (inconsistentDerivativesOnDisc(deriv)) {
-          // Set covariance to zero and return
-          // TODO: This should be changed to indicate that something went
-          // wrong
-          return ThisResult::success(std::move(nominalResult));
-        }
-      }
+
+  // Test if target is disc - this may lead to inconsistent results
+  if (target.type() == Surface::Disc) {
```

Comment:
Btw, does this not also apply to cylinder surfaces?

Reply:
Yeah you're right the dangerous area is only along the phi boundary along z, but not around the center like for a disk.